pkg/registry/core/endpoint/storage/storage_test.go

```diff
@@ -143,3 +145,64 @@ func TestWatch(t *testing.T) {
 		},
 	)
 }
+
+func TestGetK8SAlias(t *testing.T) {
+	storage, server := newStorage(t)
+	defer server.Terminate(t)
+	defer storage.Store.DestroyFunc()
+
+	ctxCreate := genericapirequest.NewDefaultContext()
+	k8s := &api.Endpoints{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "kubernetes",
+		},
+	}
+	objSaved, err := storage.Create(ctxCreate, k8s, func(obj runtime.Object) error { return nil }, &metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("failed to setup system k8s EP: %v", err)
+	}
+	k8sSaved := objSaved.(*api.Endpoints)
+
+	ctxGet := genericapirequest.NewDefaultContext()
+	ctxGet = genericapirequest.WithTenantAndNamespace(ctxGet, "bar", "default")
```

Comment:
a const for "bar"

Reply:
In test code, string literal is generally preferable to const defined in prod package